mirror/images.txt

```diff
@@ -9,12 +9,18 @@
 # ceph-csi devel
 docker.io/rook/ceph:v1.8.2	rook/ceph:v1.8.2
 
+
+# ceph-csi v3.5
+quay.io/cephcsi/ceph:v16	ceph/ceph:v16
+docker.io/rook/ceph:v1.6.2	rook/ceph:v1.6.2
+
+
 # ceph-csi v3.4
-docker.io/ceph/ceph:v16		ceph/ceph:v16
+quay.io/cephcsi/ceph:v16	ceph/ceph:v16
```

Comment:
there is no need to duplicate things, that will probably cause images to pushed to the CI mirror multiple times (and overwrites), so it just causes extra network traffic.

Reply:
sure, removed duplicated instances too @nixpanic